persist/src/main/resources/db/migration/V5__SoftDeletes.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE users
+ADD COLUMN deleted BOOLEAN NOT NULL DEFAULT FALSE;
+ALTER TABLE users
+    ADD COLUMN deleted_timestamp timestamp;
```

Comment:
⛏️  Can these be combined to one statement per table?

Reply:
I tried to put them into one statement per table, but I ran into this error: 
` Failed to execute goal org.jooq:jooq-codegen-maven:3.12.0:generate (jooq-codegen) on project persist: Error running jOOQ code generation tool: Unexpected token or missing query delimiter: [2:50] ...
[ERROR] ADD COLUMN deleted BOOLEAN NOT NULL DEFAULT FALSE[*],
[ERROR] ADD COLUMN deleted_timestamp timestamp;
[ERROR] ALTER TABLE team
`

After doing some research, it seems that one of the versions of our dependencies does not allow for multiple add column statements within an alter table statement